projects/1.21-fabric/assets/things-fabric/things/lang/zh_cn.json

```
@@ -0,0 +1,204 @@
+
+  "block.things.stone_glowstone_fixture": "石制荧石灯具",
+  "block.things.quartz_glowstone_fixture": "石英制荧石灯具",
+  "block.things.deepslate_glowstone_fixture": "深板岩制荧石灯具",
+  "block.things.gleaming_ore": "璀璨矿石",
+  "block.things.deepslate_gleaming_ore": "深层璀璨矿石",
+  "block.things.diamond_pressure_plate": "钻石压力板",
+
+  "item.things.hardening_catalyst": "硬化催化剂",
+  "item.things.hardening_catalyst.tooltip": "在铁砧中敲给任意物品让它坚不可摧",
+  "item.things.recall_potion": "回忆药水",
+  "item.things.container_key": "容器钥匙",
+  "item.things.container_key.tooltip": "潜行右键容器以上锁/解锁。可以打开用它锁上的容器",
+  "item.things.bater_wucket": "木涌",
+  "item.things.bempty_ucket": "椌甬",
+  "item.things.ender_pouch": "末影袋",
+  "item.things.ender_pouch.tooltip": [
+    {"text": "按下 ", "color": "gray"},
+    {"index": 0, "color": "white"},
+    " 打开末影箱"
+  ],
+  "item.things.monocle": "单片眼镜",
+  "item.things.monocle.tooltip": "获得永久夜视",
+  "item.things.moss_necklace": "苔藓项链",
+  "item.things.moss_necklace.tooltip": "处于光照下时获得生命恢复II",
+  "item.things.placebo": "安慰剂",
+  "item.things.placebo.tooltip": "25%概率不消耗喝下的药水",
+  "item.things.displacement_tome": "位移魔典",
+  "item.things.displacement_tome.tooltip": "便利地使用传送点。鼠标拿起时右键末影珍珠来充能",
+  "item.things.displacement_tome.tooltip.charges": [
+    {"text": "能量: ", "color": "gray"},
+    {"index": 0, "color": "#5800FF"}
+  ],
+  "item.things.displacement_page": "位移魔页",
+  "item.things.mining_gloves": "挖矿手套",
+  "item.things.mining_gloves.tooltip": "获得永久急迫II",
+  "item.things.riot_gauntlet": "狂暴臂铠",
+  "item.things.riot_gauntlet.tooltip": "获得永久力量",
+  "item.things.infernal_scepter": "烈焰魔杖",
+  "item.things.infernal_scepter.tooltip": "消耗并发射火焰弹",
+  "item.things.gleaming_powder": "璀璨粉",
+  "item.things.gleaming_compound": "璀璨合金",
+  "item.things.hades_crystal": "炼狱结晶",
+  "item.things.hades_crystal.tooltip": "获得永久抗火。同时装备 §f附魔蜡腺§7 会有更好的效果",
+  "item.things.enchanted_wax_gland": "附魔蜡腺",
+  "item.things.enchanted_wax_gland.tooltip": "让你浮在水中并游得飞快",
+  "item.things.item_magnet": "物品磁铁",
+  "item.things.item_magnet.tooltip": [
+    {"text": "⚡ ", "color": "#8000ff"},
+    {"text": "能量: ", "color": "gray"},
+    {"index": 0, "color": "dark_gray"}
+  ],
+  "item.things.consolation_cake.hint": "你可以用这个换一块安慰蛋糕",
+  "item.things.consolation_cake": [
+    {"italic": false, "text": ""},
+    {"text": "❤", "color": "#EB1D36"},
+    {"text": " 安慰蛋糕 "},
+    {"text": "❤", "color": "#EB1D36"}
+  ],
+  "item.things.rabbit_foot_charm": "兔脚护符",
+  "item.things.rabbit_foot_charm.tooltip": "获得永久跳跃提升II",
+  "item.things.luck_of_the_irish": "爱尔兰人的幸运符",
+  "item.things.luck_of_the_irish.tooltip": "让毒马铃薯有金苹果的功效",
+  "item.things.socks": "圣诞袜",
+  "item.things.socks.tooltip": [
+    "获得加速、额外行走高度和可开关的跳跃提升。 按下 ",
+    {"index": 0, "color": "white"},
+    " 开关跳跃提升"
+  ],
+  "item.things.socks.jumpy": "跳跃",
+  "item.things.socks.speed_0": "普通",
+  "item.things.socks.speed_1": "快速",
+  "item.things.socks.speed_2": "飞人博尔特",
+  "item.things.socks.speed_illegal": "非法",

```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: Naughty**

```suggestion
  "item.things.socks.speed_illegal": "非法",
```

---

错译，已修正